x-pack/plugins/alerting/server/routes/rule/apis/get/get_rule_route.ts

```diff
@@ -50,6 +50,9 @@ const buildGetRuleRoute = ({
             body: () => ruleResponseSchemaV1,
             description: 'Indicates a successful call.',
           },
+          404: {
```

Comment:
Shouldn't this also return 400 and 403?

403 because of the call to `verifyAccessAndContext` and 400 because of the call to `ensureAuthorized` in `x-pack/plugins/alerting/server/application/rule/methods/get/get_rule.ts`